crates/syntax/src/tests/sourcegen_ast.rs

```diff
@@ -259,11 +260,43 @@ fn generate_nodes(kinds: KindsSrc<'_>, grammar: &AstSrc) -> String {
                             &self.syntax
                         }
                     }
+
+                    #(
+                        impl From<#node_names> for #name {
```

Comment:
I'm not sure if those impls useful for every type and whether this is going to affect compile-time. Maybe I can create those impls for specific types?

Reply:
Ye I am not sure we gain much from this right now, so I think I'm in favor of creating the required impls manually in `node_ext`